frontend/src/WebSocketContext.jsx

```diff
@@ -19,10 +20,10 @@ export function WebSocketProvider({ children }) {
     const send = (object) => ws.current.send(JSON.stringify(object));
 
     const joinLobby = async () => {
-        const response = await fetch("http://127.0.0.1:8000/lobby/code/join", { method: "POST" });
+        const response = await fetch(`http://${BACKEND_DOMAIN}/lobby/code/join`, { method: "POST" });
         const data = await response.json();
-        send({ type: "initializer", code: "code", id: data.id });
-        send({ type: "lobby_lifecycle", lifecycle_type: "game_start" });
+        send({data: { type: "initializer", code: "code", id: data.id }});
+        send({data: { type: "lobby_lifecycle", lifecycle_type: "game_start" }});
```

Comment:
i can see that messages are being sent by the backend via logs, and via browser dev tools, but they aren't printed in the event logs. is that intentional? (if we aren't using the event log anymore, we should probably remove it)